libr/bp/bp.c

```diff
@@ -167,6 +167,9 @@ static RBreakpointItem *r_bp_add(RBreakpoint *bp, const ut8 *obytes, ut64 addr,
 	}
 	b = r_bp_item_new (bp);
 	b->addr = addr + bp->delta;
+	if (bp->baddr > addr)
+		eprint ("base addr should not be larger than the breakpoint address.");
```

Comment:
do you mean eprintf with a newline?

Reply:
why this condition? and why this condition is done before using the variables involved?

Reply:
This condition should never happen but if it does an error should be printed